osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -84,35 +86,49 @@ private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes
                 difficultyValue *= 1.025;
 
             if (score.Mods.Any(m => m is ModHardRock))
-                difficultyValue *= 1.050;
+                difficultyValue *= 1.10;
 
             if (score.Mods.Any(m => m is ModFlashlight<TaikoHitObject>))
                 difficultyValue *= 1.050 * lengthBonus;
 
-            return difficultyValue * Math.Pow(accuracy, 2.0);
+            return difficultyValue * Math.Pow(SpecialFunctions.Erf(40 / (Math.Sqrt(2) * estimatedDeviation)), 2.0);
         }
 
         private double computeAccuracyValue(ScoreInfo score, TaikoDifficultyAttributes attributes)
         {
             if (attributes.GreatHitWindow <= 0)
                 return 0;
 
-            double accuracyValue = Math.Pow(60.0 / attributes.GreatHitWindow, 1.1) * Math.Pow(accuracy, 8.0) * Math.Pow(attributes.StarRating, 0.4) * 27.0;
+            double accuracyValue = Math.Pow(7.5 / estimatedDeviation, 1.1) * Math.Pow(attributes.StarRating, 0.4) * 100.0;
```

Comment:
Should not be possible with the estimation method I'm using, if you can find a scenario where it happens I can account for it

Reply:
The scenario where it happens is where you submit an either currently impossible value (i.e. 100% accuracy and 1x miss), just needs to be clamped to prevent that from being able to be plugged into the system returning infinite pp.

Reply:
I'll return null for a greatprobability equal to or above 1 for sanity reasons then